runtime/oti/j9nonbuilder.h

```diff
@@ -3406,6 +3419,12 @@ typedef struct J9ArrayClass {
 	/* Added temporarily for consistency */
 	UDATA flattenedElementSize;
 	struct J9ClassHotFieldsInfo* hotFieldsInfo;
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	/* A linked list of weak global references to every resolved MemberName whose clazz is this class. */
+	J9MemberNameListNode *memberNames;
+	/* True if a MemberName object belonging to this class (so likely in memberNames) has been finalized. */
+	BOOLEAN needToPruneMemberNames;
```

Comment:
instead of using a whole word, I think we still have some room in clazz->classFlags